tests/unittest/CMakeLists.txt

```diff
@@ -50,6 +50,7 @@ set (CPP_LIST
     MainWindow/PerfomanceTracker_test.cpp
     MainWindow/ProjectFileComponent_test.cpp
     CFGProgrammer/CFGProgrammer_test.cpp
+    CFGProgrammer/CFGProgrammer_test.cpp
```

Comment:
This as well.

Reply:
Dropping this PR. But this issue is fixed in this new PR
https://github.com/os-fpga/FOEDAG/pull/1228/files#diff-681487157da59661329781f01f4dd79fe2ac830ef7999a40b8482893a6743b75R51
<img width="890" alt="image" src="https://github.com/os-fpga/FOEDAG/assets/115125680/2c2f89bc-7d5a-4016-a1f1-add96a3b20b2">